clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinatorTest.java

```diff
@@ -913,7 +913,8 @@ private ConsumerCoordinator buildCoordinator(Metrics metrics, List<PartitionAssi
                 retryBackoffMs,
                 defaultOffsetCommitCallback,
                 autoCommitEnabled,
-                autoCommitIntervalMs);
+                autoCommitIntervalMs,
+                null);
```

Comment:
I have tests for commits in PlaintextConsumerTest -- it tests all interceptor callbacks. I just added tests for onCommit callbacks on auto commit, since I initially missed those use-cases.